ansible/templates/config.hjson

```diff
@@ -4,15 +4,22 @@
 
   # settings related to the postgresql database
   database: {
+    database: lemmy
+    user: lemmy
+    host: postgres
+    port: 5432
+    pool_size: 5
     # password to connect to postgres
     password: "{{ postgres_password }}"
-    # host where postgres is running
-    host: "postgres"
   }
   # the domain name of your instance (eg "lemmy.ml")
   hostname: "{{ domain }}"
+  # the port
```

Comment:
"port where lemmy should listen for incoming requests"

from https://github.com/LemmyNet/lemmy/blob/main/config/defaults.hjson#L32